src/connectors/dabmusic.ts

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+Connector.playerSelector = 'div.fixed.bottom-0.left-0.right-0';
+
+Connector.getTrack = () => {
+	const trackEl = document.querySelector('div.fixed.bottom-0.left-0.right-0 h3.truncate');

```

Comment:
These seem quite brittle to any UI changes. Wonder if we could target them in a nicer way?